src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -89,6 +89,7 @@ export default (accessToken, userID, opts) => {
     ),
     savedArtworksLoader: gravityLoader("collection/saved-artwork/artworks", { user_id: userID, private: true }),
     sendFeedbackLoader: gravityLoader("feedback", {}, { method: "POST" }),
+    showLoader: gravityLoader(id => `show/${id}`),
```

Comment:
This Gravity endpoint only returns [`:public` properties](https://github.com/artsy/gravity/blob/7d49dca9bac65c88cce8b681f899a78cb1a04dcc/app/api/v1/shows_endpoint.rb#L16). It's sufficient for this particular use case, but in the future CMS is likely to need more data that's authorized for partners.

Given MP only wants to have 1 type handle the same domain model, I guess in the future we still need to either hit the [partner shows endpoint](https://github.com/artsy/gravity/blob/7d49dca9bac65c88cce8b681f899a78cb1a04dcc/app/api/v1/partners_shows_endpoint.rb#L37-L51) that currently CMS is hitting (but the extra `:partner_id` param is annoying), or consolidate the two Gravity endpoints.

Definitely not blocking for this PR and we can follow up.